pkg/commands/build/status.go

```diff
@@ -180,3 +201,49 @@ func displayBuildStatus(cmd *cobra.Command, bld v1alpha1.Build) error {
 
 	return tableWriter.Write()
 }
+
+func displayBOM(cmd *cobra.Command, bld v1alpha1.Build, rup registry.UtilProvider, tlsConfig registry.TLSConfig) error {
+	cond := bld.Status.GetCondition(corev1alpha1.ConditionSucceeded)
+	if cond == nil || !cond.IsTrue() {
+		return errors.Errorf("build has failed or has not finished")
+	}
+
+	h, err := commands.NewCommandHelper(cmd)
+	if err != nil {
+		return err
+	}
+
+	fetcher := rup.Fetcher()
+
+	image, err := fetcher.Fetch(bld.Status.LatestImage, tlsConfig)
+	if err != nil {
+		return err
+	}
+
+	c, err := image.ConfigFile()
+	if err != nil {
+		return err
+	}
+
+	bldMetadataStr, ok := c.Config.Labels[buildMetadataKey]
+	if !ok {
+		return errors.New("could not find cnb metadata on build image")
+	}
+
+	bldMetadata := map[string]interface{}{}
+	if err := json.Unmarshal([]byte(bldMetadataStr), &bldMetadata); err != nil {
+		return err
+	}
+
+	bom, ok := bldMetadata[bomKey]
+	if !ok {
+		return errors.New("could not find bom on build image metadata")
+	}
+
+	output, err := json.Marshal(bom)
```

Comment:
```suggestion
	output, err := json.MarshalIndent(bom, "", "  ")
```
does give a prettier print 🤷 
``` 	
$ kp build status test-image --bom
[
  {
    "buildpack": {
      "id": "tanzu-buildpacks/go-dist",
      "version": "0.1.2"
    },
    "metadata": {
      "licenses": [],
      "name": "Go",
      "sha256": "dd49c88e48eba4dfb33854aa4dfd205897f73998e015ee6bd0188f5d13c3ecfc",
      "stacks": [
        "io.buildpacks.stacks.bionic",
        "io.paketo.stacks.tiny",
        "org.cloudfoundry.stacks.cflinuxfs3"
      ],
      "uri": "file:///dependencies/dd49c88e48eba4dfb33854aa4dfd205897f73998e015ee6bd0188f5d13c3ecfc"
    },
    "name": "go",
    "version": "1.15.3"
  }
]
```  
vs one line wrap
```
$ kp build status test-image --bom
[{"buildpack":{"id":"tanzu-buildpacks/go-dist","version":"0.1.2"},"metadata":{"licenses":[],"name":"Go","sha256":"dd49c88e48eba4dfb33854aa4dfd205897f73998e015ee6bd0188f5d13c3ecfc","stacks":["io.buildpacks.stacks.bionic","io.paketo.stacks.tiny","org.cloudfoundry.stacks.cflinuxfs3"],"uri":"file:///dependencies/dd49c88e48eba4dfb33854aa4dfd205897f73998e015ee6bd0188f5d13c3ecfc"},"name":"go","version":"1.15.3"}]
```

Reply:
pack does a single line so, I am inclined to mat that behavior.

Reply:
looks like pack changed their behaviour
![Screen Shot 2021-02-10 at 1 50 45 PM](https://user-images.githubusercontent.com/1648613/107556963-0440d400-6ba7-11eb-97dc-cac63c32ab5c.png)